src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,14 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+  let numbertoArray = [];
+  numbertoArray = number.toString().split("").reverse();
```

Comment:
Rather than create an empty array only populate it on the next line, it could easily just be done all in one go (and be made into a `const` since it does not need reassigning):
```
  const numbertoArray = number.toString().split("").reverse();
```

I would also suggest updating the name to `numberToArray` using pascal casing